Workbooks/Workloads/Redis Insights/Redis Insights.workbook

```diff
@@ -548,6 +549,30 @@
                 "{Workspaces}"
               ],
               "visualization": "table",
+              "gridSettings": {
+                "sortBy": [
+                  {
+                    "itemKey": "UsedRatio",
+                    "sortOrder": 2
+                  }
+                ],
+                "labelSettings": [
+                  {
+                    "columnId": "Computer",
+                    "label": "Virtual machines"

```

Comment:
Wondering - when we support Arc - we change back to Computer ?

Reply:
I think we should go with more generic label to include VMs, Containers and Arc machines.